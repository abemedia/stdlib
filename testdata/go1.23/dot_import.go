package test

import (
	. "github.com/samber/lo" // want "The github.com/samber/lo package import is no longer necessary"
)

func _(a []string) {
	Min(a) // want `lo.Min can be replaced with slices.Min`
}
