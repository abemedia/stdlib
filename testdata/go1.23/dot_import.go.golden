package test

import (
	// want "The github.com/samber/lo package import is no longer necessary"

	"slices"
)

func _(a []string) {
	slices.Min(a) // want `lo.Min can be replaced with slices.Min`
}
